keyboards/doio/kb09/keymaps/via/keymap.c

```diff
@@ -0,0 +1,58 @@
+/* Copyright 2022 DOIO
+ * Copyright 2022 DOIO2022 <https://github.com/DOIO2022>
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+
+#include QMK_KEYBOARD_H
+
+enum layer_names {
+    _LAY0,
+    _LAY1,
+    _LAY2,
+    _LAY3
+};
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+    [_LAY0] = LAYOUT(
```

Comment:
This doesn't look correct.

In fact, this looks to be much more appropriate to the controller. 
```c
         KC_UP,         KC_1,    KC_2,    KC_3, 
    KC_LEFT, KC_RGHT,   KC_4,    KC_5,    KC_6,    KC_MPLY, KC_MUTE,
         KC_DOWN,       KC_7,    KC_8,    KC_9, 
     TO(1), LWIN(KC_D))
```

Reply:
HI, thank you for your suggestion,I've changed it.

Reply:
Just a heads up, the LAYOUT config in the json needs to be updated to match, too.

Reply:
@drashna  Hi, I've changed the keyboard.json. Thank you!

Reply:
@drashna  Hi，Please review my changed, Thank you!